src/page_body/structure/blocks/page_block_embed_storybook.pr

```diff
@@ -6,15 +6,24 @@
   <div class="content-block content-block--embed">
     <div class="storybook-container" style="height: {{ (block.size && block.size.height) ? block.size.height : 500 }}px">
         <div class="storybook-embed-container">
+            {* Iframe in iframe was used to prevent controlling scroll of parent window from iframe. *}
+            {* This is a hack bacause it's a Chrome bug exploit and can become broken in some time *}
+            {* More info: https://stackoverflow.com/a/71434176 *}
             <iframe
-                class="embed-responsive-item storybook"
-                src="{{ block.url }}"
-                allowfullscreen
-                loading="lazy"
-                aria-label="Embedded Storybook"
-                referrerpolicy="origin"
+                class="embed-responsive-item"
                 style="height: {{ (block.size && block.size.height && block.size.height < 150) ? block.size.height + "px" : "100%" }}"
-            ></iframe>
+                srcdoc="
+                    <iframe
```

Comment:
This is a completely new pattern to me, so I'm not able to say if it's okay or not okay to have it like this.

How confident are you that it works for all what our Storybook block supports?

Reply:
Similar issue is present on FE for cloud. So, FE team will take a look on possible solutions as well and then we are going to make the same solution for doc exporter as well.

https://linear.app/supernova-io/issue/RCT-2097/ex-docs-bug-or-preview-published-doc-when-clicking-on-the-tab-it#comment-8e5ea6a1